en/customization/introduction.md

```diff
@@ -79,7 +79,27 @@ The translation service used has the [pricing](https://azure.microsoft.com/en-us
 The price for each 1 Million characters translated is 8.43 € and there is no fixed cost per month.
 Google and DeepL have an approximate price of between 16.00 € and 20.00 € for each 1 million characters and a fixed monthly cost.
 
-Alerts can be created on several parameters, including the `Number of translated characters` within the Azure administration panel in the **Supervision** section.
+Although technical measures have been taken to prevent misuse of this service, we recommend the creation of Alerts offered by Azure so that an Administrator can be notified in the event of detecting unusual use of the service.
+
+To create an Alert in Azure we must follow the following steps:
+1. Sign in to the **Azure Portal**.
+1. Register the resource providers needed to create Alerts
+  1. On the Azure portal menu, select **All services**.
+  1. In the **All services** box, enter **subscription**, and then select **Subscriptions**.
+  1. Select the subscription from the subscription list to view.
+  1. Select **Resource providers** and view the list of available resource providers.
+  1. Among the resource providers we must select **microsoft.insights** and click on the button **Register**
+1. Create new alert rule:
+  1. On the Azure portal menu, select **All services**.
+  1. In the **All services** box, enter **subscription**, and then select **Subscriptions**.
+  1. Select the subscription from the subscription list to view.
+  1. Select **Resources** and access the resource created from translations of the type **Cognitive Services**
+  1. In the **Monitoring** section select **Alerts** and we access **New alert rule**
+  1. Select the **Resource** about which we want to add the Alert (if we have followed the previous steps it should already be selected)
```

Comment:
"about which" sounds a bit strange to me :thinking:. Maybe "on which" is better, but I'm not sure.

Since I wasn't sure, I checked the Azure docs, and rhey say: "resource that we want to alert on". So maybe we could use it :smile:.